### PR TITLE
Feature/better codecov calcs

### PR DESCRIPTION
**Pull request by khdegraaf**
Add logic to ignore certain files we can't cover, or know are covered elsewhere in code coverage calcs.  Also, some minor code coverage improvements.

### CODE REVIEW

File: cmd/ebrelayer/relayer/cosmos.go [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package relayer
 
+// DONTCOVER
```
Comment (utx0) [inline cmd/ebrelayer/relayer/cosmos.go:3]: Whats the reasoning with all the `DONTCOVER` ?

File: cmd/ebrelayer/txs/.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Replace sample private key for an active validator address
+# For local testing, use accounts[1] private key from truffle
```
Comment (utx0) [inline cmd/ebrelayer/txs/.env:2]: i'd prefer we didnt commit `.env` file into the repo so we can config things locally for iso dev/testing

File: cover.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+go test -v ./... -coverprofile=coverage.txt -covermode=atomic
```
Comment (utx0) [inline cover.sh:1]: Can we put this script in `./scripts`  :)

**Review by utx0 — COMMENTED** [review]
some comments and minor changes

File: cmd/ebrelayer/relayer/cosmos.go (hunk 1 above)
Comment (khdegraaf) [reply cmd/ebrelayer/relayer/cosmos.go:3]: The code coverage will now notice this, and excluded the file from code coverage stats.

File: cover.sh (hunk 3 above)
Comment (khdegraaf) [reply cover.sh:1]: sure

File: cmd/ebrelayer/txs/.env (hunk 2 above)
Comment (khdegraaf) [reply cmd/ebrelayer/txs/.env:2]: OK, I'll fix this.

Comment (khdegraaf) [reply cmd/ebrelayer/txs/.env]: done.

File: cover.sh (hunk 3 above)
Comment (khdegraaf) [reply cover.sh]: done.

File: cmd/ebrelayer/relayer/cosmos.go (hunk 1 above)
Comment (utx0) [reply cmd/ebrelayer/relayer/cosmos.go:3]: Sorry my question was more why are we excluding these files?

Comment (khdegraaf) [reply cmd/ebrelayer/relayer/cosmos.go:3]: These are core files in ebrelayer.  Can't mock these without significant work that is better focused elsewhere.  There are a variety of other files excluded from code coverage (via settings we copied from cosmos-sdk).  Like all top-level main.go files, as well as client directories that do boiler-plate io plumbing.  That is essentially what we are doing here.

Comment (utx0) [reply cmd/ebrelayer/relayer/cosmos.go:3]: oh ok I get it now.  Thanks

**Review by utx0 — APPROVED** [review]
Looking good.

**Pull request merged**